src/allmydata/util/eliotutil.py

```diff
@@ -87,8 +87,11 @@
 )
 from twisted.application.service import Service
 
-from .jsonbytes import AnyBytesJSONEncoder
-
+from ._eliot_updates import (
```

Comment:
Maybe document what this should be changed to, once Python 2 support is dropped? Will save a bit of time later.

Reply:
E.g. you will need to restore `capture_logging = partial(eliot.testing.capture_logging, encoder_=AnyBytesJSONEncoder)`.